app/models/page.rb

```diff
@@ -179,6 +190,13 @@ def subscriptions_count
     braintree_subscriptions.count + go_cardless_subscriptions.count
   end
 
+  # Mostly donations comes as followup action
+  # So for page which has petition and followup as donation
+  # the page is considered as petition page.
+  def donation_page?
+    plugin_names.include?('fundraiser') && !plugin_names.include?('petition')
```

Comment:
what about just using `plugins.first.is_a?(Plugins::Fundraiser)`?

Reply:
@osahyoun 
Seems promising. Will update the code

Reply:
@osahyoun 
 Is there a chance fundraiser is created first before petition plugin?